pubspec.yaml

```diff
@@ -9,3 +9,5 @@ dev_dependencies:
     sdk: flutter
   path: any
   test: ^0.12.13
+  url_launcher:
```

Comment:
Hmm, it's working great in several samples like this, I think this is the same as `any`?